docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -232,25 +237,22 @@ and then you will run the [upgrade cluster command]({{< relref "baremetal-upgrad
   ✅ validate immutable fields
   🎉 all cluster upgrade preflight validations passed
   Performing provider setup and validations
-  Pausing EKS-A cluster controller reconcile
-  Pausing Flux kustomization
-  GitOps field not specified, pause flux kustomization skipped
-  Creating bootstrap cluster
-  Installing cluster-api providers on bootstrap cluster
-  Moving cluster management from workload to bootstrap cluster
-  Upgrading workload cluster
-  Moving cluster management from bootstrap to workload cluster
-  Applying new EKS-A cluster resource; resuming reconcile
-  Resuming EKS-A controller reconciliation
+  Ensuring etcd CAPI providers exist on management cluster before upgrade
+  Pausing GitOps cluster resources reconcile
+  Upgrading core components
+  Backing up management cluster's resources before upgrading
+  Upgrading management cluster
   Updating Git Repo with new EKS-A cluster spec
-  GitOps field not specified, update git repo skipped
   Forcing reconcile Git repo with latest commit
-  GitOps not configured, force reconcile flux git repo skipped
-  Resuming Flux kustomization
-  GitOps field not specified, resume flux kustomization skipped
+  Resuming GitOps cluster resources kustomization
+  Writing cluster config file
+  🎉 Cluster upgraded!
+  Cleaning up backup resources
   ```
   
-  During the upgrade process, EKS Anywhere pauses the cluster controller reconciliation by adding the paused annotation `anywhere.eks.amazonaws.com/paused: true` to the EKS Anywhere cluster, provider datacenterconfig and machineconfig resources, before the components upgrade. After upgrade completes, the annotations are removed so that the cluster controller resumes reconciling the cluster.
+  From `eksctl anywhere` version v0.18.0 onwards, for the management cluster upgrade, the CLI relies on the EKS-A controller to upgrade all the components. If the upgrade process gets stuck and a user fixes underlying issue, the upgrade process will continue. This works even if the CLI times out during the upgrade process because the upgrade process is still active since it runs in the controller.
+
+  During the workload cluster upgrade process, EKS Anywhere pauses the cluster controller reconciliation by adding the paused annotation `anywhere.eks.amazonaws.com/paused: true` to the EKS Anywhere cluster, provider datacenterconfig and machineconfig resources, before the components upgrade. After upgrade completes, the annotations are removed so that the cluster controller resumes reconciling the cluster. If the CLI execution is interrupted or times out, the controller won't reconcile changes to the EKS-A objects until these annotations are removed. You can re-run the CLI to restart the upgrade process or remove the annotations manually with `kubectl`.
```

Comment:
Workload cluster upgrade process using CLI still follows CLI code and doesn't rely on EKS-A controller. This sentence is already there on the [doc site](https://anywhere.eks.amazonaws.com/docs/clustermgmt/cluster-upgrades/baremetal-upgrades/#output), I have just differentiated it for the workload cluster since we have changed the flow only for the management cluster for now.

Reply:
Ah of course, thanks for clarifying.